multiuser/machine-auth/che-multiuser-machine-authentication/src/main/java/org/eclipse/che/multiuser/machine/authentication/server/MachineTokenPermissionsFilter.java

```diff
@@ -1,59 +0,0 @@
-/*
- * Copyright (c) 2012-2017 Red Hat, Inc.
- * All rights reserved. This program and the accompanying materials
- * are made available under the terms of the Eclipse Public License v1.0
- * which accompanies this distribution, and is available at
- * http://www.eclipse.org/legal/epl-v10.html
- *
- * Contributors:
- *   Red Hat, Inc. - initial API and implementation
- */
-package org.eclipse.che.multiuser.machine.authentication.server;
-
-import static org.eclipse.che.multiuser.permission.workspace.server.WorkspaceDomain.DOMAIN_ID;
-import static org.eclipse.che.multiuser.permission.workspace.server.WorkspaceDomain.USE;
-
-import javax.ws.rs.Path;
-import org.eclipse.che.api.core.ApiException;
-import org.eclipse.che.api.core.ForbiddenException;
-import org.eclipse.che.commons.env.EnvironmentContext;
-import org.eclipse.che.commons.subject.Subject;
-import org.eclipse.che.everrest.CheMethodInvokerFilter;
-import org.everrest.core.Filter;
-import org.everrest.core.resource.GenericResourceMethod;
-
-/**
- * Restricts access to methods of {@link MachineTokenService} by user's permissions
- *
- * @author Max Shaposhnik (mshaposhnik@codenvy.com)
- */
-@Filter
-@Path("/machine/token{path:(/.*)?}")
-public class MachineTokenPermissionsFilter extends CheMethodInvokerFilter {
```

Comment:
Looks like `MachineTokenService` still exists, but not protected. Can we remove it or protect?

Reply:
Forgot to remove, fixed